sections/calibration_products.tex

```diff
@@ -17,7 +16,7 @@ \subsection{Key Requirements}
 The work performed in this WBS serves several complementary roles:
 
 \begin{itemize}
- \item It will enable the production of calibration data products as required by the Level 2 Photometric Calibration Plan (\NewPCP{}) and other planning documents \cite{Lupton15}. This includes both characterization of the sensitivity of the LSST system (optics, filters and detector) and the transmissivity of, and emission from, the atmosphere;
+ \item It will enable the production of calibration data products as required by the Level 2 Photometric Calibration Plan (\NewPCP{}) and other planning documents. This includes both characterization of the sensitivity of the LSST system (optics, filters and detector) and the transmissivity of, and emission from, the atmosphere;
```

Comment:
I think I'd drop "and other planning documents.", otherwise I wonder what those documents might be.